.github/components.yml

```diff
@@ -70,6 +71,7 @@ AUTO_BATCH:
     - TEMPLATE
     - C_API
     - Python_API
+    - JS_API    
```

Comment:
how HETERO or AUTO BATCH affects JS?
IMO, those all are not required

Reply:
maybe you misunderstands how smart CI works.
when AUTO_BATCH is changed, current PR forces to run JS API tests, why do we need it? how AUTO BATCH can break JS API?

Reply:
as we were instructed adding an area to the list **excludes** i.e JS API from AUTO plugin tests.
will ping you internally about it

Reply:
`  revalidate:` is not the same as `exclude` :D

Reply:
yes, my bad :D 
in that case I'm opting for removing it from this section